keen/explorer
# Fixes #137

## What does this PR do?
Fixes #137. Sets the "saving" property back to false after an attempted save when the model is invalid.

## How can I test it?

1) Fill out the query builder with a new query. Leave the "Query name" field above the visualization area blank.
2) Click "save". Observe that there is an error message saying that you must fill out the query name.
3) Click "run query".
4) Observe that the query successfully ran and the error message was cleared.
